src/directives/formBuilderDnd.js

```diff
@@ -31,7 +31,7 @@ module.exports = [
 
     $scope.addComponent = function(component, index) {
       // Only edit immediately for components that are not resource comps.
-      if (component.isNew && (!component.key || (component.key.indexOf('.') === -1))) {
+      if (component.isNew && !component.isNonEditable && (!component.key || (component.key.indexOf('.') === -1))) {
```

Comment:
Actually I was trying to come up with a fix to this issue: https://github.com/formio/ngFormBuilder/issues/212
The component could be edited later if the user clicks edit in the form-builder. This is just to stop the ngDialog from being opened when the component is added. What do you suggest for the property name?